template/apps/web/.eslintrc.js

```diff
@@ -10,8 +10,9 @@ module.exports = {
     'next',
     'airbnb',
     'airbnb-typescript',
+    'plugin:storybook/recommended'
   ],
-  ignorePatterns: ['.eslintrc.js'],
+  ignorePatterns: ['.eslintrc.js', '!.storybook'],
```

Comment:
I don't know how to do it differently, but I read that eslint automatically adds .storybook to ignore and it needs to be removed from ignore using !.